src/sections/location/location-table-row.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState, useCallback } from 'react';
+
+import Box from '@mui/material/Box';
+import Avatar from '@mui/material/Avatar';
+import Popover from '@mui/material/Popover';
+import TableRow from '@mui/material/TableRow';
+import Checkbox from '@mui/material/Checkbox';
+import MenuList from '@mui/material/MenuList';
+import TableCell from '@mui/material/TableCell';
+import IconButton from '@mui/material/IconButton';
+import MenuItem, { menuItemClasses } from '@mui/material/MenuItem';
+
+import { Label } from 'src/components/label';
+import { Iconify } from 'src/components/iconify';
+
+// ----------------------------------------------------------------------
+
+export type UserProps = {
+  id: string;
+  name: string;
+  role: string;
+  status: string;
+  company: string;
+  avatarUrl: string;
+  isVerified: boolean;
+};
```

Comment:
if these are to display user details, then instead of this take reference from updated code and use redux states, defined to store user info.

Reply:
Will be resolved in future updates